container/hello/Dockerfile

```diff
@@ -41,7 +41,8 @@ FROM amazonlinux:2 as image
 # Copying dependencies of the enclave apps from the 'full_image'
 # to shrink the final image size.
 #
-COPY --from=full_image /ne-deps /
+RUN mkdir -p ne-deps
+COPY --from=full_image /ne-deps /ne-deps
```

Comment:
Sorry, but I don't understand how this should work.
We've created a shrinked root fs in `/ne-deps` with assumption to copy it all together to a target container root folder. IIRC it [didn't work](https://github.com/aws/aws-nitro-enclaves-with-k8s/issues/35) because something changed in how `/lib64` is represented (now it's a link) in the latest AL.

But now if we copy `/ne-deps` to the target `/ne-deps` how all the binaries are found if you just call `nitro-cli` in `run.sh`?

Reply:
I believe what you are talking about is happening earlier and we are in fact making a deep copy of necessary libraries in lines 20-26 by using the `-L` option in the copying.